lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -943,37 +943,123 @@ class ConvertAtenConvolutionOp : public OpConversionPattern<AtenConvolutionOp> {
     // - grouped 1d-3d (quantized)
     // - ungrouped 1d-3d
     if (groupSize == 1 && !inputZp && !weightZp) {
+      // Get default accumulator tensor type for the result tensor and use
+      // it for upcast/downcast if it doesn't match with the result type.
+      Value accumulatorTensor;
+      Type accumulatorTensorType = getDefaultAccType(rewriter, resultDTy);
```

Comment:
You should not be calling this `TensorType` as it is an element type of the tensor. `accumulatorDtype` is more correct.